crates/types/src/node_id.rs

```diff
@@ -125,6 +125,12 @@ impl GenerationalNodeId {
 #[display("N{}", _0)]
 pub struct PlainNodeId(u32);
 
+impl PlainNodeId {
+    pub fn raw_id(self) -> u32 {
+        self.0
+    }
```

Comment:
u32::from(plain_node_id) should do the same. So there is no need to add `raw_id`.